### PR TITLE
Qt/486 with dummy toolchain

### PR DESCRIPTION
**Pull request by jgphpc**
```== COMPLETED: Installation ended successfully``` on ela
Please review.
( where should i push the easyconfig file ? )

### CODE REVIEW

File: easybuild/easyblocks/q/qt.py [hunk 1]
```diff
@@ -41,18 +42,32 @@ class EB_Qt(ConfigureMake):
...
         """Configure Qt using interactive `configure` script."""
 
         self.cfg.update('configopts', '-release')
 
+        platform = None
         comp_fam = self.toolchain.comp_family()
-        if comp_fam in [toolchain.GCC]:  #@UndefinedVariable
-            self.cfg.update('configopts', '-platform linux-g++-64')
+        if self.cfg['platform']:
+                platform = self.cfg['platform']
```
Comment (boegel) [inline easybuild/easyblocks/q/qt.py]: @jgphpc the indentation is off here, each indent level should be 4 spaces (same below)

**Comment by boegel** [comment]
Jenkins: ok to test

@jgphpc Are you issuing a PR for an easyconfig that requires these changes too?

**Comment by boegel** [comment]
@jgphpc This looks excellent now, thanks!

I'd love to have an PR for a Qt easyconfig files that requires these changes...
Is that something you plan on tackling as well?

**Comment by jgphpc** [comment]
@boegel  sure: https://github.com/hpcugent/easybuild-easyconfigs/pull/2821
( I also did https://github.com/hpcugent/easybuild-easyconfigs/pull/2820 (master instead of develop) by mistake... )

**Comment by boegel** [comment]
Tested with https://github.com/hpcugent/easybuild-easyconfigs/pull/2821 and existing Qt easyconfigs, good to go!

Thanks @jgphpc!

**Pull request merged**